docs/ECLLanguageReference/ECLR_mods/Value-Data.xml

```diff
@@ -24,6 +24,9 @@
   role="bold">Working with BLOBs</emphasis> for more information on this
   subject.</para>
 
+  <para>The upper size limit for any DATA value is 4GB (2^32), although you
```

Comment:
Actually the limits are probably 2^32-17!
I wouldn't document that, but it also may not be worth saying 2^32.